ledger/ledger-api-test-tool/src/main/scala/com/daml/ledger/api/testtool/tests/ContractKeys.scala

```diff
@@ -162,10 +163,22 @@ final class ContractKeys(session: LedgerSession) extends LedgerTestSuite(session
       }
     }
 
+  val recreateContractKeys =
+    LedgerTest("CKRecreate", "Contract keys can be recreated in single transaction") { context =>
+      val key = s"${UUID.randomUUID.toString}-key"
+      for {
+        ledger <- context.participant()
+        Vector(owner) <- ledger.allocateParties(1)
```

Comment:
You can use `ledger.allocateParty()` to only get back a single one.

Reply:
Which would be:
```suggestion
        owner <- ledger.allocateParty()
```

Reply:
Ah, already done, sorry, just noticed.